src/server/template-renderer/index.js

```diff
@@ -191,10 +191,13 @@ export default class TemplateRenderer {
       contextKey = 'state',
       windowKey = '__INITIAL_STATE__'
     } = options || {}
+    const autoRemove = process.env.NODE_ENV === 'production'
+      ? 'var s;(s=document.currentScript||document.scripts[document.scripts.length-1]).parentNode.removeChild(s);'
```

Comment:
`document.currentScript` is not supported in IE yet, hence the fallback.

Reply:
This block should be wrapped in an IIFE to avoid leaking `s` to global scope.

Reply:
Fixed in https://github.com/vuejs/vue/pull/6763/commits/2409ff0e2feb4a465ed7eecbf9dc4e58547908b8